omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -57,10 +66,13 @@
     <% end -%>
 
     <% if node['private_chef']['data_collector']['root_url'] && node['private_chef']['data_collector']['token'] -%>
-    location ~ /organizations/([^/]+)/data-collector {
+    location ~ "^/organizations/([^/]+)/data-collector$" {
```

Comment:
we post to `/data-collector/v0`, IIRC, we also do a GET to that endpoint to check that it's there. If I'm reading this correctly, this will break posts to `/v0` and will also prevent GETs as well